spec/components/external_links/hathi_links_component.html.erb_spec.rb

```diff
@@ -4,37 +4,69 @@
 
 RSpec.describe ExternalLinks::HathiLinksComponent, type: :component do
   subject(:rendered) do
-    render_inline(described_class.new(links: hathi_links,
-                                      show_links: show_hathi_links))
```

Comment:
@ajkiessl I might not be understanding correctly so ignore my comment if your changes are already working the way I am mentionin here: we probably wanna leave the show_hathi_links checks in place, it should be working the same way when show_hathi_links is true and update the spec for when show_hathi_links false.

Reply:
I think you're right.  I was a little confused with that at first, but I see this is used in the BaseLinksComponent that HathiLinksComponent inherits from.  I'll change this.